docs/Panorama/event-system.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Event System
+parent: Panorama
+nav_order: 100
+---
+
+# Event System
+Panorama uses an event system to communicate between the engine and panels but also between panels. They can be defined and subscribed to via Javascript.
+
+## Defining Events
+```
+**TODO** What is the difference between these two functions
```

Comment:
Never actually used `$.DefinePanelEvent` but panel events are things like `onmouseover`, `onactivate` etc... No idea why you'd want define custom ones from JS tbh.

Worth distinguishing the two though. Regular events are basically like you say below. Panel events are subscribed to using `panel.SetPanelEvent(event name, callback)`